### PR TITLE
feat(docs): create page on publishing data

### PR DESCRIPTION
**Pull request by MaximilianLombardo**
- #TICKET_NUMBER

### Reviewers
**Functional:** 
@brianraymor 
@ambrosejcarr 

**Readability:** 

---

## Changes
- add page on publishing data

### CODE REVIEW

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
+  - drug screens 
+- recieve confirmation form the curation team will accept your dataset along with instructions on how to format your dataset for submission.
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:12]: RE _recieve confirmation form the curation team will_ - 

1. recieve -> receive 
2. form -> from ?

Is this intended to be something like _If your request is approved, the curation team will respond with further instructions on how to format ...._

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- Prepare your dataset and email our curators for further assistance in the process
+- Publish!
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:14]: This is missing some iterations between the curations and the data submitter. There is a review. The submitter approves and asks the curators to publish.

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Contributing Data
+
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:8]: _We do not allow submission_ -> _We do not accept datasets_

Please be consistent about _dataset_ vs _data set_.

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  - ethnicity_ontology_term _id: HANCESTRO if human, ‘unknown’ if info unavailable, ‘na’ if non-human
+  - development_stage_ontology_term _id: HsapDv if human, MmusDv if mouse, ‘unknown’ if info unavailable
+  - sex_ontology_term_id: PATO:0000384 for male, PATO:0000383 for female, or ‘unknown’ if unavailable
+- **Embeddings in obsm**:
+  - One or more two-dimensional embeddings, prefixed with 'X_'
+- **Features in var & raw.var (if present)**:
+  - index is Ensembl ID
+  - preference is that gene have not been filtered in order to maximize future data integration efforts
+
+
```
Comment (brianraymor) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:55]: This is missing the data submission policy?

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  - schema_version: '2.0.0'
+  - title: title of the individual dataset visualization
+  - X_normalization: method used to normalize the data stored in X and any scaling or logging; 'none' if data in X is raw 
+  - optional: batch_condition: list of obs fields that define that “batches” that a normalization or integration algorithm should be aware of
+- **Data in .X and raw.X**:
+  - raw counts are required
+  - normalized counts are strongly recommended
+  - raw counts should be in raw.X if normalized counts are in .X
+  - if there is no normalized matrix, raw counts should be in .X
+- **Cells metadata in obs (the most specific term available from the specified ontology)**:
```
Comment (jahilton) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:40]: I think we should try to make this more clear or we'll get a bunch of 'scRNA-seq' data from 'adult'
**Cells metadata in ob - the values MUST be the most specific term available from the specified ontology**
(See schema for guidance and examples)

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 4 above)
Comment (jahilton) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:55]: +1

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
+  - drug screens 
+- recieve confirmation form the curation team will accept your dataset along with instructions on how to format your dataset for submission.
+- Prepare your dataset and email our curators for further assistance in the process
+- Publish!
+
+
```
[the remark this answers is not in the document]
Comment (jahilton) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:16]: Rethinking the link to the validation tool. If we did that, we'd also need to include specification for only genes in the pinned annotation, possibly more specs.
So thinking that validation is part of our curation tools, not part of their submission tools...

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 2 above)
Comment (ambrosejcarr) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:14]: This section conflates data eligibility and data publishing, which has created confusion. We should put eligibility criteria into "requirements", as indicated in the first paragraph, and only talk about the submission process here. 

I've addressed the formatting/spelling (!) issues above with my suggestion. 

I discuss the eligibility criteria in a separate comment, below. 

```suggestion
## Publishing Process

The process for submission to the portal is:
- You confirm we can support your submission by reaching out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the data that you'd like to contribute.
- We confirm that we will accept your data.
- You prepare your data according to our submission [requirements](#dataset-requirements) and send us your files.
- We upload to a private collection where you can review.
- We publish the data when you tell us to.
```

@jahilton does this text reflect how you'd like to see the process described?

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+## Dataset Requirements
+
```
Comment (ambrosejcarr) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:19]: I suggest we add an eligibility section here, to match my two formatting suggestions above. 

```suggestion
## Dataset Requirements

### Data Eligibility

Cellxgene is focused on supporting the global community attempting to create references of human cells and tissues. As a result, there are a few kinds of datasets that we are **not** likely to accept at this time:
- drug screens
- cell lines
- organisms other than mouse or human 

### Formatting Requirements

```

@jahilton you're correct that we haven't held a hard line on any of these three criteria so we shouldn't *prohibit*, but I do want to give submitters a sense of what might be rejected so we don't set up incorrect expectations. 

For cell lines, I regret the covid infected cells collection. The explorer interface is built for identifying discrete cell types and is not great for measuring subtle changes in phenotype introduced by drug screens/perturbations. If we wanted to support those data types in the future, I think we'd seriously consider building new interfaces.

**Review by ambrosejcarr — CHANGES_REQUESTED** [review]
I've suggested a restructuring which attempts to fix a few of @jahilton and @brianraymor comments. They should review and determine if I succeeded.

Please fix the remainder of their suggested edits.

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 1 above)
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:12]: edited the pr to address both of these comments

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 8]
```diff
@@ -0,0 +1,56 @@
+# Contributing Data
+
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
```
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:10]: I'll add an item that says needs further review - I was going off of this discussion: https://czi-sci.slack.com/archives/C037WS42XV3/p1654569348667679?thread_ts=1654274960.404139&cid=C037WS42XV3

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:8]: rewrote this sentence to be clearer

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:16]: My understanding is that the purpose of this page is just to give a general sense of the curation process and that once the authors contact us, then they'll get additional relevant information

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+We need the following collection metadata (i.e. details associate with your publication or study)
+
+- Collection information:
+  - Title
+  - Description
+  - Contact: name and email
+  - Publication/preprint DOI: can be added later
+  - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
+ 
+Datasets need the following information added to a single h5ad (AnnData) format file per cellxgene visualization:
```
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:29]: added

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 7 above)
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:19]: add in my new amendment to the commit

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 4 above)
Comment (brianraymor) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:66]: _direct personal identifiers_ is a link to the information in - https://github.com/chanzuckerberg/single-cell-data-portal/blob/41af03dd63690210a245c00c05b38989a1769207/frontend/src/components/Collections/components/PublishCollection/components/Policy/components/DirectPersonalIdentifiers/index.tsx

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 7 above)
Comment (jahilton) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:17]: The softer language makes sense to me, as is the need to steer away from these data types.

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+Cellxgene supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you have single cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The process for submission to the portal is:
+- You confirm we can support your submission by reaching out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the data that you'd like to contribute.
+- We confirm that we will accept your data.
+- You prepare your data according to our submission [requirements](#dataset-requirements) and send us your files.
+- We upload to a private collection where you can review.
+- We publish the data when you tell us to.
```
Comment (jahilton) [inline frontend/doc-site/032__Contribute and Publish Data.mdx:12]: I suggest adding "You prepare revised data and send us your revised files, as needed." before the publish step.

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (jahilton) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:14]: I'm not sure where the line is, but it feels like we're somewhere in between at the moment
One extreme would be "Here's the eligibility criteria. If you fit, reach out for more instructions". Then they reach out & re respond, as we do now
Right now, it is that plus "...and here's some info for how to prep your data...but not all of it"

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 10 above)
Comment (ambrosejcarr) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:12]:
```suggestion
- You prepare revised data and send us your revised files, as needed.
- We publish the data when you tell us to.
```

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+- Collection information:
+  - Title
+  - Description
+  - Contact: name and email
+  - Publication/preprint DOI: can be added later
+  - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
+ 
+Datasets need the following information added to a single h5ad (AnnData 0.7) format file per cellxgene visualization:
+- **Dataset-level metadata in uns**:
+  - schema_version: '2.0.0'
```
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:38]: addressed

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+The process for submission to the portal is:
+- You confirm we can support your submission by reaching out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the data that you'd like to contribute.
+- We confirm that we will accept your data.
+- You prepare your data according to our submission [requirements](#dataset-requirements) and send us your files.
+- We upload to a private collection where you can review.
+- We publish the data when you tell us to.
+
+
+
+## Dataset Requirements
```
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:17]: updated

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+  - schema_version: '2.0.0'
+  - title: title of the individual dataset visualization
+  - X_normalization: method used to normalize the data stored in X and any scaling or logging; 'none' if data in X is raw 
+  - optional: batch_condition: list of obs fields that define that “batches” that a normalization or integration algorithm should be aware of
+- **Data in .X and raw.X**:
+  - raw counts are required
+  - normalized counts are strongly recommended
+  - raw counts should be in raw.X if normalized counts are in .X
+  - if there is no normalized matrix, raw counts should be in .X
+- **Cell metadata in obs (the values MUST be the most specific specific term available from the specified ontology)**:
```
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:47]: Added 'Let us know when the most specific term available isn't an accurate descriptor of your metadata, so we can flag these cases for updates in the future'

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 2 above)
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx]: @jahilton

File: frontend/doc-site/032__Contribute and Publish Data.mdx [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
+  - drug screens
```
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx]: addressed in later changes

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (MaximilianLombardo) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:15]: Can you give me a more specific example?

[the remark this answers is not in the document]
Comment (ambrosejcarr) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:15]: > We should note that they are welcome to submit as much additional metadata as they'd like, but should avoid redundancy.
> And link to the full schema specs

Added in 8c4d720d

> And that the ontology labels and gene symbols will be filled in upon submission so submitting those in separate fields would be redundant.

Left this one out, couldn't find a way to incorporate without TMI.

File: frontend/doc-site/032__Contribute and Publish Data.mdx (hunk 4 above)
Comment (ambrosejcarr) [reply frontend/doc-site/032__Contribute and Publish Data.mdx:67]: > direct personal identifiers is a link to the information in - https://github.com/chanzuckerberg/single-cell-data-portal/blob/41af03dd63690210a245c00c05b38989a1769207/frontend/src/components/Collections/components/PublishCollection/components/Policy/components/DirectPersonalIdentifiers/index.tsx

Added in 5a46c3da

**Review by ambrosejcarr — APPROVED** [review]
I've reviewed the open threads and suggested text to address them. I'm going to merge this PR so that we can get it released before HCA. @jahilton if you have additional language changes to suggest, we would welcome an additional PR!

**Pull request merged**